### PR TITLE
Add request duration panels to Matrix dashboard

### PR DESCRIPTION
**Pull request by calvinnhieu**

### CODE REVIEW

**Review by parthshahva — CHANGES_REQUESTED** [review]
@calvinnhieu please move fogg upgrade into a separate PR -----> Ignore this comment. You force pushed this change after I had already looked at the PR 👍

File: terraform/modules/env-dashboards/matrix-dashboard.tf [hunk 1]
```diff
@@ -18,7 +18,8 @@ locals {
   "editable": true,
   "gnetId": null,
   "graphTooltip": 0,
-  "iteration": 1543885117046,
+  "id": 65,
```
Comment (parthshahva) [inline terraform/modules/env-dashboards/matrix-dashboard.tf]: Remove id field. It should already be templated elsewhere in the json

File: terraform/modules/env-dashboards/matrix-dashboard.tf [hunk 2]
```diff
@@ -2805,6 +2806,796 @@ locals {
...
+          "dimensions": {
+            "Number of Bundles": "0-499",
+            "Output Format": "zarr"
+          },
+          "highResolution": false,
+          "metricName": "Matrix Request Duration",
+          "namespace": "dcp-matrix-service-${var.env}",
+          "period": "",
+          "refId": "A",
+          "region": "default",
```
Comment (parthshahva) [inline terraform/modules/env-dashboards/matrix-dashboard.tf]: please template the region as well

Comment (calvinnhieu) [reply terraform/modules/env-dashboards/matrix-dashboard.tf]: Good catch, missed these "default"s

**Comment by parthshahva** [comment]
@calvinnhieu have you tried deploying this via the deployment tooling?

**Comment by calvinnhieu** [comment]
No, wasn't sure if I should try deploying before or after changes were reviewed

**Review by parthshahva — APPROVED** [review]

**Comment by parthshahva** [comment]
Feel free to merge after you've confirmed the deployment to another environment works as expected

**Pull request merged**